tests/e2e/target-aggregates.spec.js

```diff
@@ -102,6 +102,7 @@ const updateSettings = (targetsConfig, user, contactSummary) => {
 const clickOnTargetAggregateListItem = (contactId) => {
   element(by.css(`.aggregate-detail li[data-record-id="${contactId}"] a`)).click();
   helper.waitUntilReady(element(by.id('contacts-list')));
+  helper.waitUntilReady(element(by.css('.content-pane .meta h2')));
```

Comment:
In hindsight, I don't think this actually de-flakes the test. I'll check this out again.

Reply:
I changed the approach around de-flaking the unit test so it waits for the contact-summary to be loaded.